nibabel/freesurfer/tests/test_io.py

```diff
@@ -156,17 +156,17 @@ def test_write_morph_data():
         for shape in okay_shapes:
             write_morph_data('test.curv', values.reshape(shape))
             # Check ordering is preserved, regardless of shape
-            assert_equal(values, read_morph_data('test.curv'))
-        assert_raises(ValueError, write_morph_data, 'test.curv',
-                      np.zeros(shape), big_num)
-        # Windows 32-bit overflows Python int
+            assert np.array_equal(read_morph_data('test.curv') ,values)
+        with pytest.raises(ValueError):
+            write_morph_data('test.curv', np.zeros(shape), big_num)
+                # Windows 32-bit overflows Python int
```

Comment:
Looks like some overzealous tabbing.

```suggestion
        # Windows 32-bit overflows Python int
```